### ISSUE TITLE
Not able to connect to dummy device

### ISSUE THREAD

**Issue by iqtidar27** [DigitalSecurity/btlejuice#57]
Hello, 
I was able to run the webserver and showed nearby ble devices:

![image](https://user-images.githubusercontent.com/59121751/71639297-f699ea80-2c41-11ea-8e0e-da356bf06475.png)
![image](https://user-images.githubusercontent.com/59121751/71639301-09142400-2c42-11ea-91d9-77acb778f03b.png)
Btlejuice created the dummy device for the pulse oximeter but when my oximeter application device was trying to connect to the dummy device its always showed "connecting"

For the Quardioarm it was always trying to make the dummy device but was never successful.

![image](https://user-images.githubusercontent.com/59121751/71639321-745df600-2c42-11ea-9e1f-9d9a2784dd87.png)

Does anyone face the same issue? I would really appreciate any suggestion here.

**Comment by ItIsMe4894** [comment]
Mind telling us how it was solved?

**Comment by iqtidar27** [comment]
It was a trial and error process. I was trying to connect BtleJuice webserver with the Qardioarm android app. So, instead of turning Bluetooth on directly from my android settings, I turned it off initially. When I open the APP it asked me to turn on the Bluetooth, then I turn it on. Finally, it works. Still, it works 60% of the time. Sometimes I had to restart my device to clean up the Bluetooth cash.
